packages/nextjs/components/scaffold-stark/Input/InputBase.tsx

```diff
@@ -62,7 +62,7 @@ export const InputBase = <
     <div className={`flex bg-input text-accent ${modifier}`}>
       {prefix}
       <input
-        className="input input-ghost focus-within:border-transparent focus:outline-none focus:bg-transparent h-[2.2rem] min-h-[2.2rem] px-4 border w-full text-xs placeholder:text-[#9596BF] text-neutral"
+        className="input input-ghost focus-within:border-transparent focus:outline-none focus:bg-transparent h-[2.2rem] min-h-[2.2rem] px-4 border w-full text-xs placeholder:text-[#9596BF] text-neutral rounded-none"
         placeholder={placeholder}
         name={name}
         value={value?.toString()}
```

Comment:
When switch between undefined and some value, it may throw warning about `controlled component` and `uncontrolled component`, for our usage, it is an controlled component, better to have a default value like this: `value?.toString() || ""`

Reply:
will fix! thanks for pointing that out!